### PR TITLE
VTS monitor and VTF monitor

### PR DESCRIPTION
**Pull request by nikolay-fedotov**

### CODE REVIEW

File: lab/vts.py [hunk 1]
```diff
@@ -137,15 +160,53 @@ def json_api_session(self):
...
         finally:
             s.close()
-        return r.json()
+        return r
 
     def json_api_get_network_inventory(self):
         return self.json_api_get('rs/ncs/query/networkInventory')
+
+    def get_overlay_networks(self):
+        return self.json_api_get('rs/ncs/query/topologiesNetworkAll?limit=2147483647&name=admin')
```
Comment (kshileev) [inline lab/vts.py]: admin -> should be parameter

File: lab/vts.py [hunk 2]
```diff
@@ -65,9 +65,32 @@ def actuate(self, proxy):
             file_name = self._proxy_to_run.put_string_as_file_in_dir(string_to_put=tmpl, file_name=file_name)
             self._commands[cmd] = file_name
 
+    def _get(self, raw, key):
```
Comment (kshileev) [inline lab/vts.py]: isn't re.find_all() a substitution for this _get?

File: lab/scenarios/vts_scenario.py [hunk 3]
```diff
@@ -14,13 +14,35 @@ def loop(self):
...
         for i in range(1, self._n_instances):
             port_pids = self._cloud.create_ports(instance_name=str(i), on_nets=internal_nets, is_fixed_ip=True)
             instance_name = self._cloud.create_instance(name=str(i), flavor='m1.medium', image=image_name, on_ports=port_pids)
 
+            ports += port_pids
+
             # cloud.cmd('ping -c5 {fip}'.format(fip=fips[i]))
             # instances[i] = Server(ip=fips[i], username='root', lab=None, name='a')
 
+        vtc1 = self._cloud.mediator.lab().get_node('vtc1')
```
Comment (kshileev) [inline lab/scenarios/vts_scenario.py]: their might be a delay between port-binding oper in OS and time when those ports will be visible in VTC
also, it's good to have it as a helper in Vts class- like: 
Vts.check_port_id(port_id)

File: lab/scenarios/vts_scenario.py [hunk 4]
```diff
@@ -14,13 +14,35 @@ def loop(self):
...
+        for name, internal_net in internal_nets.items():
```
Comment (kshileev) [inline lab/scenarios/vts_scenario.py]: why in separate cycle?

File: lab/vts.py (hunk 1 above)
Comment (nikolay-fedotov) [inline lab/vts.py]: 1. Done
2. limit is a max integer value.

**Pull request merged**